### DISCUSSION TITLE
Ідеї для покращення

### DISCUSSION THREAD

**Discussion by MaksymKurny** [MaksymKurny/UpdAter#1] [Ideas]
Пишіть)

**Comment by Trucker-69** [comment]
Чувак, дякую за таку програму, але додай будь ласка пошук, у мене 150+ українізаторів в одній програмі, проблематично з пошуком потрібної гри. Дякую

**Reply by MaksymKurny** [reply]
Радий чути, спробую додати

**Reply by Trucker-69** [reply]
дякую, за оновлення
